src/main/java/net/sf/jabref/model/database/DuplicationChecker.java

```diff
@@ -16,14 +16,22 @@
     // use a map instead of a set since i need to know how many of each key is in there
     private final Map<String, Integer> allKeys = new HashMap<>();
 
-    //##########################################
-    //  usage:
-    //  isDuplicate=checkForDuplicateKeyAndAdd( null, b.getKey() , issueDuplicateWarning);
-    //############################################
-    // if the newkey already exists and is not the same as oldkey it will give a warning
-    // else it will add the newkey to the to set and remove the oldkey
+
+    /**
+     * Usage:
+     * isDuplicate=checkForDuplicateKeyAndAdd( null, b.getKey() , issueDuplicateWarning);
```

Comment:
This will be formatted as "Usage: isDuplicate=-......". If you really want a new line, use html elements such as `<br>` or `<p>` in the javadoc comments.